modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/setDataRecord.es.js

```diff
@@ -49,16 +49,6 @@ export default (
 
 		let availableLanguageIds;
 
-		Object.keys(localizedValue)
```

Comment:
Hey, @adolfopa. This can't be removed for now. Here's a scenario where your change will impact:

1. Create a structure with more than one translation;
2. Create a web content, fill the fields using the default language;
3. Then select one the languages the structure was translated to.

You'll notice that the field's default value wasn't carried to the translation. The value is reset, but the expected behavior is to maintain the default value.